### ISSUE TITLE
Getting a configuration ignoring cache

### ISSUE THREAD

**Issue by bvisonl** [craue/CraueConfigBundle#45]
I am not sure if maybe it is me doing something wrong but I have the DoctrineCacheBundleAdapter setup and so far it is working like a charm. However, what happens if in a very specific section I want the configuration bundle to fetch a configuration from the database ignoring the cache?

An example on this is that there is a value that I am constantly changing directly on the database to test something out but since the result is cached it is obviously not working properly.

**Comment by craue** [comment]
Well, you could remove the cache from your dev env and use it in staging/prod only.

**Comment by bvisonl** [comment]
Yeah but it is more like if there is a value that is constantly changing (let's say that it changes directly in the database from another app that connects to the database..) and you want only that configuration to be loaded ignoring cache
